src/loader.prefix

```diff
@@ -7,5 +7,24 @@
 (function() {
     function isFunction(value) {return typeof value === 'function';}
     function isDefined(value) {return typeof value !== 'undefined';}
+    function isNumber(value) {return typeof value === 'number';}
     function isObject(value) {return value !== null && typeof value === 'object';}
+    function isScope(obj) {return obj && obj.$evalAsync && obj.$watch;}
+    function isWindow(obj) {return obj && obj.window === obj;}
+    function sliceArgs(args, startIndex) {return Array.prototype.slice.call(args, startIndex || 0);}
+    function toJsonReplacer(key, value) {
```

Comment:
In this file - a note at the top is probably enough, saying something of the effect that these functions are defined here because they must be available in angular-loader without angular.js loaded, and that they are copied from the Angular.js file